src/main/java/com/codeup/adlister/controllers/LoginServlet.java

```diff
@@ -18,19 +18,17 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
             response.sendRedirect("/profile");
             return;
         }
-        request.getRequestDispatcher("/WEB-INF/login.jsp").forward(request, response);
+        request.getRequestDispatcher("/WEB-INF/user/login.jsp").forward(request, response);
     }
 
     protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
         String username = request.getParameter("username");
         String password = request.getParameter("password");
         User user = DaoFactory.getUsersDao().findByUsername(username);
-        System.out.println("test");
+
         if (user == null || !BCrypt.checkpw(password, user.getPassword())) {
-            System.out.println("inside if statment");
```

Comment:
Thank you for removing the `sout`s

Reply:
I'm still not entirely sure on merges 😬 Would it be possible to have a review or example of how we should collaborating before Capstone? Thank you so much for looking over!

Reply:
Sure, no problem! 😄